tests/e2e/framework/rawVM.go

```diff
@@ -211,7 +211,7 @@ func (vm *GCPRawVM) provision() error {
 			 --can-ip-forward \
 			 --service-account %s \
 			 --scopes "https://www.googleapis.com/auth/cloud-platform" \
-			 --tags "http-server","https-bootstrap" \
+			 --tags "http-server","https-server" \
```

Comment:
How did this pass the tests in the first place?

Reply:
that's sadly one of the "yet to be integrated" test (which also why mesh expansion broke in 0.3.0 etc...)